docs/eigenlayer/operator-guides/key-management/intro-key-management.md

```diff
@@ -9,21 +9,38 @@ This section presents key management suggestions for operators. Within the conte
+Central to every Proof of Stake mechanism lies a signature scheme. Signatures serve to authenticate the identity of every validator, enabling the attribution of their actions, whether positive or negative, to them. A validator's honesty can be confirmed by examining their signed messages, while malicious behavior can be demonstrated through messages that contravene consensus rules.
 
-A validator's honesty can be confirmed by examining their signed messages, while malicious behavior can be demonstrated through messages that contravene consensus rules.
+### Ethereum
 
-Indeed, in Ethereum, a validator's identity is synonymous with their public key. To be precise, each validator possesses two sets: a signing and withdrawal keys.
+Focusing on Ethereum validator keys exemplifies keys necessitating optimal security and accessibility measures, as they may maintain a robust connection with Nodes, further underlining their importance. Indeed, in Ethereum, a validator's identity is synonymous with their public key. To be precise, each validator possesses two sets: a signing and withdrawal keys.
 
-### Signing keys
+#### Signing keys
 
 A *signing key* is the key a validator needs to sign attestations and propose blocks. Because a validator needs to sign a message at least once per epoch, the client software must have custody of the key.
 
-### Withdrawal keys
+#### Withdrawal keys
 
 Because the client software is always connected to the internet, there is a chance that one’s signing key is compromised. To reduce the impact of such a breach, the actions a validator can perform are split between two keys.
 
 The signing key, as explained above, is used for the validator to perform their duties. On the other hand, the *withdrawal key* has the power to control a validator's funds (transferring and withdrawing ETH).
 
 A validator should only need to use their withdrawal keys a few times over the lifetime of being a validator. This means they can be put into cold storage and stored with high security (offline).
 
+### Eigenlayer
+
+Eigenlayer works differently from Ethereum due to it being natively a delegated PoS system. This means operators don't need to custody a separate key for each 32 Eth; they only need a single key per eigenlayer operator. This does imply that proper key management tools are even more important.
+
+Contrary to Eigenlayer, where an operator controls a bunch of (signing, withdrawal) key pairs, an eigenlayer operator controls a single operator key, as well as a separate key (or a small number of keys) for each AVS they are registered in.
+
+#### Operator keys
+
+An operator's operator key is the ecdsa key that is used to authenticate any interaction with the eigenlayer core contracts, and hence control an operator's actions such as registering to an operator, changing its operator parameters, force undelegating a staker, etc.
```

Comment:
`force undelegating a staker` - do we have this now?

Reply:
yes, see https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/docs/core/DelegationManager.md#undelegate
Do you want me to link to this section as well?

Reply:
nah this is okay